libs/filter-form/src/lib/filter-form/filter-form.component.ts

```diff
@@ -25,33 +26,42 @@ export class FilterFormComponent implements OnInit {
   @Output()
   formUpdate = new EventEmitter<NotificationsFilter>();
 
-  selectedCities = new Set([]);
-
   @Input()
-  set cityList(citiesList: Map<string, string>) {
-    this.#cities = citiesList;
-    this.filterCities('');
+  set cityList(cityList_: Map<string, string>) {
+    this.cityList_ = cityList_;
   }
   get cityList() {
-    return this.#cities;
+    return this.cityList_;
+  }
```

Comment:
If all the `setter/getter` do is set and get a value - there's no real need of a `getter/setter`.
```
@Input() cityList: Map<string,string>;
```
Would do the same. Don't you think?